scipy/sparse/csgraph/_laplacian.py

```diff
@@ -109,8 +134,12 @@ def _laplacian_sparse(graph, normed=False, axis=0):
     return m, w
 
 
-def _laplacian_dense(graph, normed=False, axis=0):
-    m = np.array(graph)
+def _laplacian_dense(graph, normed=False, axis=0,
+                     inplace=False):
+    if type(graph) is np.ndarray and inplace:
+        m = graph
+    else:
+        m = np.array(graph)
```

Comment:
Why not just
```suggestion
    m = np.asarray(graph)
```

Reply:
This is the only place where `inplace` is used. If deleted, `inplace` would just have no effect?

Reply:
Would need some tests, but `asarray` should not do anything if it's already an array (the case here). To me it should be equivalent.

Reply:
Your proposed edit would just get the original code back. Since the original is not inplace, I guess that m = np.array(graph) creates a copy if graph is already an array. I'll try to test it.

Reply:
My edit vs the original code is `asarray` instead of `array`

Reply:
Sorry, I missed `as`. You are correct. Testing it now.

Reply:
But I am also correct, since your edit removes any use of `inplace', so it has no effect.

Reply:
The correct version should be:
```
    if inplace:
            m = np.asarray(graph)
        else:
            m = np.array(graph)
```
Testing the above now

Reply:
Correct 👍 Just not sure about the `asarray` because it will not be inplace if there is a conversion to make (not a ndarray).